### ISSUE TITLE
[feature] Assign servers the names of Indigogo donors.

### ISSUE THREAD

**Issue by myleshorton** [getlantern/lantern#2061]

**Comment by atavism** [comment]
@myleshorton, there's a corresponding ticket for the client-side changes? I presume to display the hostnames for donor servers instead of IP addresses when you hover over the peer icons, right?

**Comment by myleshorton** [comment]
Just created it @atavism -- #2190 thanks!

**Comment by Derekf5** [comment]
@myleshorton @atavism here are the names of the servers that we have received thus far.

tychometadata
Creativity
hypnagogic
elesdyzer
merfolk
SHOWSTORM 
Hi’iaka 
Hecate

**Comment by myleshorton** [comment]
Thanks!

On Tuesday, February 10, 2015, Derekf5 <notifications@github.com> wrote:

> @myleshorton <https://github.com/myleshorton> @atavism
> <https://github.com/atavism> here are the names of the servers that we
> have received thus far.
>
> tychometadata
> Creativity
> hypnagogic
> elesdyzer
> merfolk
> SHOWSTORM
> Hi’iaka
> Hecate
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/lantern/issues/2061#issuecomment-73785463>.
>


-- 
--
President
Brave New Software Project, Inc.
https://www.getlantern.org
pgp A998 2B6E EF1C 373E 723F A813 045D A255 901A FD89

**Comment by myleshorton** [comment]
Hey @aranhoide let's try to just add this to our static IP distribution. How's that sound?

**Comment by aranhoide** [comment]
SGTM.  At the very least a fp- prefix will need to be prepended.  So, it's just a matter of making their host name / instance ID match that, right?

Would Hi-iaka be an acceptable sanitization of Hi'iaka for the sake of becoming a hostname?

**Comment by myleshorton** [comment]
Yeah I think it is just a matter of including those chosen names somewhere in the name/instance ID. There will undoubtedly them be some client side work to parse out the names and display them, but we can handle that separately. So ideally the names continue to include any useful information, such as the data center they're in, but then also include these custom names in these few cases.

**Comment by atavism** [comment]
@myleshorton There's already a ticket for the client-side changes https://github.com/getlantern/lantern/issues/2061 I added it to the Beta 5 milestone as well.

**Comment by myleshorton** [comment]
The server side of this is done right @aranhoide?

**Comment by aranhoide** [comment]
IIRC, last we talked, there needs not be a server side to this.  If we have
a list of donor names we can just assign them to servers in genconfig.
